ofrak_core/ofrak/ofrak_cli.py

```diff
@@ -270,16 +282,69 @@ def handler(ofrak_env: OFRAKEnvironment, args: Namespace):
 
             dependency_info = f"{dep.tool}\n\t{dep.tool_homepage}\n\t[{', '.join(c for c in components_by_dep[dep])}]"
             if not args.no_check:
-                dependency_info = f"[{' ' if not is_installed else '✓'}] " + dependency_info
+                dependency_info = (
+                    f"[{' ' if not is_installed else '✓'}] " + dependency_info
+                )
 
             print(dependency_info)
 
 
+class GUISubCommand(OFRAKSubCommand):
+    def create_parser(self, ofrak_subparsers):
+        gui_parser = ofrak_subparsers.add_parser(
+            "gui",
+            help="Launch the OFRAK GUI server.",
+            description="Launch the OFRAK GUI server.",
+        )
+        gui_parser.add_argument(
+            "-H",
+            "--hostname",
+            action="store",
+            help="Set GUI server host address.",
+            default=None,
+        )
+        gui_parser.add_argument(
+            "-p",
+            "--port",
+            action="store",
+            help="Set GUI server host port.",
+            default=None,
+        )
+        return gui_parser
+
+    @staticmethod
+    def handler(ofrak_env: OFRAKEnvironment, args: Namespace):
+        if args.hostname is not None:
+            host = args.hostname
+        else:
+            host = "127.0.0.1"
+
+        if args.port is not None:
+            port = args.port
+        else:
+            port = "8080"
+
+        ofrak = server.OFRAK(server.logging.INFO)
+
+        server.LOGGER.warning(
+            "No disassembler backend specified, so no disassembly will be possible"
+        )
+
+        url = "http://" + host + ":" + port
```

Comment:
Typically, throughout the codebase, we try to use [Python's f-strings](https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python) when possible.

```suggestion
        url = f"http://{host}:{port}"
```